src/main/java/com/github/manolo8/darkbot/gui/MapDrawer.java

```diff
@@ -158,13 +160,25 @@ public void setup(Main main) {
         this.ships = main.mapManager.entities.ships;
         this.battleStations = main.mapManager.entities.battleStations;
         this.basePoints = main.mapManager.entities.basePoints;
+
+        try {
+            image = UIUtils.getImage("background");
+        }catch (Exception e){
+            System.out.println("Background image file not found! Ignore if not set");
+        }
+
     }
 
     @Override
     protected void paintComponent(Graphics g) {
         if (main == null) return;
         Graphics2D g2 = setupDraw(g);
 
+        if(main.config.BOT_SETTINGS.MAP_DISPLAY.CUSTOM_BACKGROUND && image != null){
+            g2.setComposite(AlphaComposite.getInstance(AlphaComposite.SRC_OVER, 0.3f));
```

Comment:
It'd be cool to be able to configure that 0.3f as a transparency parameter in the config, but it's fine if it stays like this too

Reply:
that is really good idea, i added that in